examples/models/llama3_2_vision/text_decoder/model.py

```diff
@@ -142,13 +142,13 @@ def __init__(self, **kwargs):
 
             self.model_ = prune_output_vocab(self.model_, output_prune_map)
 
-        if self.use_kv_cache:
-            print("Setting up KV cache on the model...")
-            self.model_.setup_caches(
-                batch_size=1,
-                dtype=self.dtype,
-                decoder_max_seq_len=self.max_seq_len,
-            )
+        # if self.use_kv_cache:
+        #     print("Setting up KV cache on the model...")
+        #     self.model_.setup_caches(
+        #         batch_size=1,
+        #         dtype=self.dtype,
+        #         decoder_max_seq_len=self.max_seq_len,
+        #     )
```

Comment:
Need to do this because source transform happens after the model is set up, and we need to call the new swapped-in ET attention's setup_cache function. So we move the setup_cache to after the source transform